src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -168,12 +168,8 @@ def test_vm_image_list_offers(self):
 
 class VMImageListPublishersScenarioTest(ScenarioTest):
 
+    @AllowLargeResponse()
```

Comment:
I think there are several other instances where this is should be applied. Do you plan to update the remaining instances?

Reply:
Done